src/main/java/chatbot/DeadlineTask.java

```diff
@@ -0,0 +1,28 @@
+package chatbot;
+
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import chatbot.exceptions.InvalidArgumentException;
+
+class DeadlineTask extends Task implements Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
Will it be better if you caps it since its final? Otherwise, UID should not be in caps as well! But I do like how descriptive your constant is :)